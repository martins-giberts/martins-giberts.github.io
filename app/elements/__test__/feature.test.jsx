/* globals describe, it, expect */

import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import Component from '../feature/index'

describe('elements/Feature test', () => {
  it('should render defaults without throwing an error', () => {
    const wrapper = shallow(<Component name={'Test Feature'} link={'#'} text={'Test text'} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
