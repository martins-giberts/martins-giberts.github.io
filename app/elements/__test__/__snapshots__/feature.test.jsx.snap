// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/Feature test should render defaults without throwing an error 1`] = `
<div
  className="feature"
>
  <div
    className="link"
  >
    <Link
      href="#"
    >
      <i
        aria-hidden="true"
        className="fa fa-Test Feature"
      />
    </Link>
  </div>
  <div
    className="text"
    dangerouslySetInnerHTML={
      Object {
        "__html": "Test text",
      }
    }
  />
  <div
    className="button"
  >
    <Link
      href="#"
    >
      Open
    </Link>
  </div>
</div>
`;
